hdk/virtual_machine/parser.py

```diff
@@ -0,0 +1,82 @@
+"""Functions for parsing symbolic instructions into their underlying fields."""
+import re
+
+from hdk.virtual_machine.syntax import (
+    Instruction,
+    MemoryAccessInstruction,
+    StackInstruction,
+)
+
+
+def remove_comment(line: str) -> str:
+    """Removes any comments from a given line of vm source code.
+
+    Args:
+        line: A line of code that may contain a comment.
+
+    Returns:
+        The line of code with any comment removed.
+
+    Typical usage example:
+        >>> remove_comment('// The whole line is a comment')
+        ''
+        >>> remove_comment('Command // comment')
+        'Command '
+    """
+    return line.partition("//")[0]
+
+
+def remove_whitespaces(line: str) -> str:
```

Comment:
Remove this function. You don't need it. The split function will take care of different kind of whitespaces.